src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTabbedPaneUI.java

```diff
@@ -652,10 +652,11 @@ private void paintTab(SynthContext ss, Graphics g,
                     tabRect, iconRect, textRect, isSelected);
             clippedTitle = SwingUtilities2.clipStringIfNecessary(null, metrics,
                            title, textRect.width);
+            paintIcon(g, tabPlacement, tabIndex, icon, iconRect, isSelected);
             paintText(ss, g, tabPlacement, font, metrics,
                     tabIndex, clippedTitle, textRect, isSelected);
 
-            paintIcon(g, tabPlacement, tabIndex, icon, iconRect, isSelected);
+
```

Comment:
What's the purpose of moving this line? Doesn't seem to change anything from what I've tested so far.

Reply:
@prsadhuk Is there any change in rendering based on what we draw first?